distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-fanout-framework-contents.adoc

```diff
@@ -0,0 +1,270 @@
+
+==== Catalog Fanout Framework 
+
+The Fanout Catalog Framework (`fanout-catalogframework` bundle) provides an implementation of the Catalog Framework that acts as a proxy, federating requests to all available sources. 
```

Comment:
We need to update this section. The `fanout` is currently not a bundle, but a configuration property on the Standard Catalog Framework.